File: Problem/pset1/mario/mario.c
//ヘッダー
#include <cs50.h>
#include <stdio.h>

//プロトタイプ宣言
int Height_int(void) ;

int main(void)
{   
    int Height = Height_int() ;
    for (int i = 0; i < Height; i++)
  {
      for (int k = 0; k < Height - i- 1; k++)
      {
        printf(" ");
      }
      
      
      for (int j = 0; j < i + 1; j++)
      {   
        printf("#");
      }
      
      printf("\n");
  }

}


int Height_int(void) //高さの入力
{
  int n;
  do
   {
    n = get_int("Height number") ;
   }
  while (n < 1 || n > 8) ;
  return n;
}

File: Problem/pset6/mario/mario.py
from cs50 import get_int


def main():
    
    h = Height_int()
    for i in range(h):
        
        for k in range(h - i - 1):
            print(" ", end="")
            k = k + 1
        
        for j in range(i + 1):
            print("#", end="")
            j = j + 1
        
        print()
    
    i = i + 1
    
        
def Height_int():
    while (True):
        n = get_int("Height number")
        if n > 0 and n < 9:
            break
    return n


main()

